resources/js/Shared/Components/InfoPopup.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+import { ref } from 'vue'
+import { HeartIcon as OutlineHeartIcon } from '@heroicons/vue/24/outline'
+
+const showDescription = ref(false)
+</script>
+
+<template>
+  <div @mouseover="showDescription = true" @mouseout="showDescription = false">
+    <span><OutlineHeartIcon class="h-6 w-6 text-gray-500" /></span>
+    <div v-show="showDescription" class="description">
+      Login or create account to add to favorites
+    </div>
+  </div>
+</template>
+
+<style>
```

Comment:
Please convert this style to predefined [Tailwind CSS](https://tailwindcss.com/) classes.

Reply:
Converted